lib/screens/create_region_event_screens/interactor/mappers/change_date_state_mapper.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/create_region_event_screens/interactor/viewmodels/create_region_event_bloc.dart';
+
+class ChangeDateStateMapper extends StateMapper {
+  CreateRegionEventState mapResultToState(CreateRegionEventState currentState, DateTime eventDate) {
+    DateTime selectedDate;
+
+    //If time is already selected, create a new DateTime with the current date
+    if (currentState.eventTime != null) {
+      selectedDate = DateTime(
+          eventDate.year, eventDate.month, eventDate.day, currentState.eventTime!.hour, currentState.eventTime!.minute);
```

Comment:
comma at the end might make this look better

Reply:
added 👍